setup.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier:	ISC
+
+import os
+from setuptools import setup
+
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+setup(name='stpm',
+      version='0.1',
+      description='SymbiFlow Tools Package Manager',
+      url='http://github.com/antmicro/symbiflow-tools-package-manager',
+      author='Antmicro',
+      author_email='jkowalewski@antmicro.com',
```

Comment:
I would change this to symbiflow authors probably

```suggestion
      author='SymbiFlow Authors',
      author_email='symbiflow@lists.librecores.org',
```